test/video.test.js

```diff
@@ -103,6 +105,17 @@ describe('Video', () => {
 			video.videoEl.className.should.equal('class-one class-two o-video__video');
 		});
 
+		it('should support the playsinline option', () => {
```

Comment:
I added the missing attribute check above that, to `should add a video element` - but good point, I'll add an explicit test for what happens when false is passed in.